Update README.md

jt-nti opened the pull request:
Add archived warning

jt-nti committed:
Update README.md

Add archived warning

Signed-off-by: James Taylor <jamest@uk.ibm.com>

PhilWakelin commented:
If this is superseded by a user agent allow list, can we update the readme to clarify this.

jt-nti committed:
Update README.md

Add link to the "Defining a client allowlist to CMCI JVM server in a
CICSplex" topic

Signed-off-by: James Taylor <jamest@uk.ibm.com>

jt-nti commented:
@PhilWakelin good idea...

https://github.com/cicsdev/cics-explorer-access-level-policing/pull/1/commits/113b8b56ac74ba5627ea9222a438b0a49cb7d71d
